Pass `external` to esbuild in the Cloudflare adapters (#10424)
sveltejs/kit: .github/PULL_REQUEST_TEMPLATE.md

- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets

> - [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts.
>       Changeset messages should generally be prefixed with `feat:` or `fix:`.
> 	  PRs can also be prefixed with `chore:` or `docs:`, which typically won't require a changeset as they're not user visible.

Reviewer: let's put this on one line
```suggestion
- [ ] Run `pnpm changeset` for user-visible changes and follow the prompts. Changeset messages should generally be prefixed with `feat:` or `fix:`. PRs can also be prefixed with `chore:` or `docs:`, which typically won't require a changeset as they're not user visible.
```

> - [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts.
>       Changeset messages should generally be prefixed with `feat:` or `fix:`.
>       PRs can also be prefixed with `chore:` or `docs:`, which typically won't require a changeset as they're not user visible.

Reviewer: let's put this on one line
```suggestion
- [ ] Run `pnpm changeset` for user-visible changes and follow the prompts. Changeset messages should generally be prefixed with `feat:` or `fix:`. PRs can also be prefixed with `chore:` or `docs:`, which typically won't require a changeset as they're not user-visible.
```

Author: This should now be applied. My last comment and your last suggestions crossed each other 🙈
